mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go.erb

```diff
@@ -1512,6 +1512,32 @@ func TestAccSqlDatabaseInstance_rootPasswordShouldBeUpdatable(t *testing.T) {
 	})
 }
 
+func TestAccSqlDatabaseInstance_activationPolicy(t *testing.T) {
```

Comment:
It looks like this test could replace `TestAccSqlDatabaseInstance_mysqlMajorVersionUpgrade` - sql tests are long and expensive so this could be a good opportunity to consolidate, even if it's just two tests.

Reply:
Done